packages/react-native-editor/ios/Podfile.lock

```diff
@@ -457,7 +457,7 @@ SPEC CHECKSUMS:
   BVLinearGradient: 2c791e973a3df0df46028210c530fde52c06b717
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   FBLazyVector: 49cbe4b43e445b06bf29199b6ad2057649e4c8f5
-  FBReactNativeSpec: 80e9cf1155002ee4720084d8813326d913815e2f
+  FBReactNativeSpec: 483e4a3c2982c1ce57d08e90fea6be852dd5f691
```

Comment:
Noting that I had to run `npm run bundle`, which generated a new `FBReactNativeSpec` hash, before I could get this to work on my local setup. I had originally thought that this would just work automatically after checking out the branches. A beginner's mistake on my side, no doubt, but it may be worth adding this as a step in the testing instructions.

Reply:
I don't think bundling should be needed. Could you please confirm if you were testing this in the Gutenberg Mobile demo app or in the main WPiOS app?

Reply:
Aha, sorry about that! I got mixed up while trying a myriad of setups to see the changes on my side. 🤦‍♀️ As you explained via Slack, I now see that the hash is automatically generated when running the demo from within the `gutenberg-mobile` file. So... Nothing to see here... 😅 I'll mark this thread as resolved as it's a non-issue. :)

Reply:
No worries, thanks for checking though 👍